frontends/ol-util/src/lib/utils.ts

```diff
@@ -17,41 +19,12 @@ export const initials = (title: string): string => {
 export const capitalize = (txt: string) =>
   (txt[0] ?? "").toUpperCase() + txt.slice(1).toLowerCase()
 
-export const PHONE = "PHONE"
-export const TABLET = "TABLET"
-export const DESKTOP = "DESKTOP"
-
-export const PHONE_WIDTH = 599
-export const TABLET_WIDTH = 999
-
-export const getViewportWidth = () => window.innerWidth
-
-export const useWidth = () => {
-  const [width, setWidth] = useState(getViewportWidth())
-
-  useEffect(() => {
-    const cb = () => {
-      setWidth(getViewportWidth())
-    }
-    window.addEventListener("resize", cb)
-    return () => {
-      window.removeEventListener("resize", cb)
-    }
-  }, [])
-
-  return width
-}
-
-export const useDeviceCategory = () => {
```

Comment:
In Open discussions, the `PHONE_WIDTH` and `TABLET_WIDTH` values are different from our breakpoint values used in CSS.

`useMuiBreakpoint` will help us be more consistent.